include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -47,6 +47,12 @@ static const char* const kOrtSessionOptionsConfigSetDenormalAsZero = "session.se
 // Its default value is "0"
 static const char* const kOrtSessionOptionsDisableQuantQDQ = "session.disable_quant_qdq";
 
+// It controls whether to enable Double QDQ remover and Indentical Children Consolidation
+// "0": enable. ORT does remove the middle 2 Nodes from a Q->(QD->Q)->QD pairs
+// "1": disable. ORT doesn't remove the middle 2 Nodes from a Q->(QD->Q)->QD pairs
+// Its default value is "0"
+static const char* const kOrtSessionOptionsEnableDoubleQDQRemover = "session.enable_double_qdq_remover";
```

Comment:
naming: do you mean disable_double_qdq_remover?

Reply:
Either way, it is enabled by default, one can set enabled = 0 to disable it.